karaf/features/src/main/feature/feature.xml

```diff
@@ -276,60 +276,6 @@
         <feature>jclouds-api-filesystem</feature>
 
         <bundle start-level="85">mvn:org.apache.brooklyn/brooklyn-locations-jclouds/${project.version}</bundle>
-        <feature>brooklyn-software-winrm</feature>
```

Comment:
`winrm` is part of core. Don't see where do you include the feature back (in -library and -dist PRs)?